app/controllers/my_facilities_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class MyFacilitiesController < AdminController
+  include DistrictFiltering
+  include Pagination
+  include SizeFiltering
+
+  before_action :authorize_my_facilities, only: %i[index ranked_facilities blood_pressure_control]
```

Comment:
I'd say it's likely that any additional actions in this controller would also need to be authorized the same. Can we drop the `only` clause?